pkg/k8s/eventrecorder.go

```diff
@@ -21,4 +21,6 @@ const (
 	CreateFailed = "CreateFailed"
 	// CreateSucceeded is an event provided via EventRecorder
 	CreateSucceeded = "CreateSucceeded"
+	// DeleteSucceeded is an event provided via EventRecorder
+	DeleteSucceeded = "DeleteSucceeded"
```

Comment:
It might make sense to re-use kubernetes' statuses (Created, Started, etc.):
```
Events:
  Type    Reason     Age   From               Message
  ----    ------     ----  ----               -------
  Normal  Scheduled  7s    default-scheduler  Successfully assigned kubestone/iperf3-sample-889d7cfcb-2vsvp to fish
  Normal  Pulled     6s    kubelet, denali    Container image "xridge/iperf3:3.7.0" already present on machine
  Normal  Created    5s    kubelet, denali    Created container server
  Normal  Started    5s    kubelet, denali    Started container server
```

Reply:
Created a new issue to track this: #1